src/contentscripts/jobs/situational_awareness.js

```diff
@@ -192,6 +201,114 @@ function requestPowerOutagesLayerUpdate() {
 }
 
 /**
+ * Requests an update to the SES teams location layer.
+ */
+function requestSesTeamsLayerUpdate() {
+    console.debug('updating SES teams layer');
+
+    // Get all units
+    let unit = [];
+    let host = location.origin;
+
+    // Grab the last 24 hours
+    let start = new Date();
+    start.setDate(start.getDate() - 1);
+    let end = new Date();
+
+    LighthouseTeam.get_teams(unit, host, start, end, token, function(teams) {
```

Comment:
Looks like this pulls all teams, then walks them to find active ones. We should probably expand the get_teams function to take advantage of the ability of the API to filter on the search so we dont have to return so much data